.github/workflows/label.yml

```diff
@@ -0,0 +1,16 @@
+name: PR-Labeler
+on: 
+  pull_request:
```

Comment:
This needs to be `pull_request_target` to be able to access secrets when opened from a fork.

Reply:
Is this a security risk? If someone creates a pr that changes this workflow to use a custom image that snoops secrets, would we pass the secret into that image?

Reply:
`pull_request_target` runs using the workflow code of the master branch and not the PR branch afaik.